src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1938,9 +1922,25 @@ namespace WaterToAirHeatPumpSimple {
 		SimpleWatertoAirHP( HPNum ).PartLoadRatio = PartLoadRatio;
 		SimpleWatertoAirHP( HPNum ).AirMassFlowRate = PLRCorrLoadSideMdot;
 
-		SimpleWatertoAirHP( HPNum ).WaterMassFlowRate = SourceSideMassFlowRate;
-		SimpleWatertoAirHP( HPNum ).OutletWaterTemp = SourceSideInletTemp + QSource / ( SourceSideMassFlowRate * CpWater );
-		SimpleWatertoAirHP( HPNum ).OutletWaterEnthalpy = SourceSideInletEnth + QSource / SourceSideMassFlowRate;
+		if ( ( SimpleWatertoAirHP( HPNum ).WaterCyclingMode ) == WaterCycling ) {
+			SimpleWatertoAirHP( HPNum ).WaterMassFlowRate = SourceSideMassFlowRate * PartLoadRatio;
+			SetComponentFlowRate( SimpleWatertoAirHP( HPNum ).WaterMassFlowRate, SimpleWatertoAirHP( HPNum ).WaterInletNodeNum, SimpleWatertoAirHP( HPNum ).WaterOutletNodeNum, SimpleWatertoAirHP( HPNum ).LoopNum, SimpleWatertoAirHP( HPNum ).LoopSide, SimpleWatertoAirHP( HPNum ).BranchNum, SimpleWatertoAirHP( HPNum ).CompNum );
```

Comment:
Set the new water side flow rate. This is the average water flow over the time step and does not affect full load performance calculated above. For cycling models, the fluid flow is averaged over the time step and outlet node properties denote full capacity. For constant models, the fluid flow is full flow over the time step and outlet node properties are averaged over the time step.